### ISSUE TITLE
Let users choose their own colors

### ISSUE THREAD

**Issue by nmashton** [caktus/chippy#29]
When planning sprints with the physical board, we generally prefer this or that color (I'm red, for example). We should be able to choose these colors. It's also inconvenient when two automatically chosen colors are very close to one another (right now, Gerald and I are two very similar shades of pink).

**Comment by tobiasmcnulty** [comment]
Good idea! A workaround in the short term is to change the spelling of your name slightly (e.g., add a last initial), which can be kinda fun to see what color you end up with.

**Comment by gannon-h** [comment]
+1
